.yo-rc.json

```diff
@@ -1,18 +1,17 @@
 {
   "generator-jhipster": {
     "promptValues": {
-      "packageName": "com.icthh.xm.ms.scheduler",
-      "nativeLanguage": "en"
+      "packageName": "com.icthh.xm.ms.scheduler"
     },
-    "jhipsterVersion": "4.13.2",
+    "jhipsterVersion": "5.7.2",
+    "applicationType": "microservice",
     "baseName": "scheduler",
     "packageName": "com.icthh.xm.ms.scheduler",
     "packageFolder": "com/icthh/xm/ms/scheduler",
-    "serverPort": "8081",
+    "serverPort": "8086",
```

Comment:
why do we need to change port?

Reply:
according to the
https://github.com/xm-online/xm-online/wiki/Ports-reservation-map

8086 | Microservice for Scheduler | xm-ms-scheduler
-- | -- | --